samples/helloworld-agent/src/main/java/com/example/application/ImageOutBase64ResponseAgent.java

```diff
@@ -0,0 +1,25 @@
+package com.example.application;
+
+import akka.javasdk.agent.Agent;
+import akka.javasdk.agent.ResponseContent;
+import akka.javasdk.annotations.Component;
+import akka.runtime.sdk.spi.SpiAgent;
+
+// tag::class[]
+@Component(id = "image-out-base64")
+public class ImageOutBase64ResponseAgent extends Agent {
+
+  public Effect<String> image() {
+    return effects()
+      .systemMessage("testing")
+      .userMessage("generate nice images of cats and dogs")
+      .responseAsMultiModal()
+      .map(multi -> multi.contents().stream().filter(c -> SpiAgent.ImageResponseContent.class.isAssignableFrom(c.getClass()))
```

Comment:
why is this looking for SPI class here? is that wrong?

Reply:
yes, it was a previous implementation before introducing `multiModalMapping`, fixed now